qwiic_i2c/circuitpy_i2c.py

```diff
@@ -110,7 +110,10 @@ def __init__(self):
 	def isPlatform(cls):
 
 		# circuit py is mostly - for our purposes - on the samd21 or samd51 based boards
-		return os.uname().sysname in ('samd21', 'samd51')
+		try:
+			return os.uname().sysname in ('samd21', 'samd51')
+		except AttributeError:
+			return False
```

Comment:
FYI this can also be solved by just not installing the CircuitPython (and Linux) I2C files, but will leave this since it's not hurting anything.